src/Systmm.f90

```diff
@@ -126,6 +126,7 @@ SUBROUTINE SYSTMM(temp_file)
 ! allocate(temp_out(4))
 temp_out = 10
 allocate (trib_res(heat_cells))
+[CONSIDER ADDING A SUBROUTINE THAT INTIALIZES VARIABLES LIKE:T_epil, T_hyp, K_z, etc - JRY]
```

Comment:
So in the main program (rbm10_VIC.f90), call a subroutine that initializes all the global parameters in the Systmm.f90 subroutine?